salesagility/SuiteCRM
# Fix for Issue #8220

## Description




See Issue #8220 

## Motivation and Context

Formatting for readability and differentiating alert levels.

## How To Test This

Apply this PR, and check out your alerts.
For some test code to create alerts, see https://gist.github.com/ApatheticCosmos/72d75b9bf2fde915e1f82d9b2906edfa

## Types of changes


### Final checklist
